_rules/line-height-not-important-78fd32.md

```diff
@@ -21,32 +21,35 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element that is [visible][] and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
+This rule applies to any HTML element with [visible][] text that includes a [soft wrap break](https://www.w3.org/TR/css-text-3/#soft-wrap-break) and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
```

Comment:
Should we have viewport width to test the line break?
(maybe extracting "multi-line text" into a separate definition)

Notably, Inapplicable example 3 does contain a line break if the viewport is small enough…

Reply:
Are you suggesting that we specify a view port size that should be used when testing with the rule? Is that something that has been done before, I tried to cheat through this by specifying widths for specific elements so it would automatically have a line break.

Agreed its trick with the inapplicable example, we can't really make an example like that without a line break because the viewport can always get smaller. I could add an `overflow: scroll` container around it, though thats basically an anti-pattern. E.g.,
```html
    <div style="overflow-x: scroll;">
      <p style="line-height: 1em !important; width: 1000px;">
        The toy brought back fond memories of being lost in the rain forest.
      </p>
    </div>
```

Reply:
I did the above modification so that we have an inapplicable example where changing the view port size does not cause a line break to occur.

Reply:
I was more thinking something like 
"This rule applies to any element with text that includes a line break at viewport size of X by Y"
with a note, likely in Accessibility Support or Assumption, that "X by Y" is arbitrarily but based on some measurements used elsewhere in WCAG and stuff like that (and assumed that WCAG needs good line spacing at least at that viewport size).

We do have a similar trick in [Zoomed text node is not clipped with CSS overflow)(https://act-rules.github.io/rules/59br37).

Reply:
I was hesitant to include an arbitrary port side which is why I decided to fix the inapplicable example. It makes me feel a bit better that there is precedent for using the viewport size in the CSS overflow rule. That rule hasn't been viewed by TF yet and I am expecting that this will raise a discussion there, which I plan to bring up this week to get ahead of the problem.